CHANGELOG.md

```diff
@@ -2,21 +2,53 @@
 
 ## Unreleased
```

Comment:
we should call out the cold start fix here as well, similar to https://github.com/getsentry/sentry-dart/pull/2786/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR75 so ppl seeing the warning in the previous releases can easily determine that this release has the fix